quadrat/functions.php

```diff
@@ -19,6 +19,7 @@ function quadrat_support() {
 				'social'  => __( 'Social Links Navigation', 'quadrat' ),
 			)
 		);
+		remove_theme_support( 'block-templates' );
```

Comment:
Why do we need this?

Reply:
I don't think we do. I suspect this comes from a rebase I didn't do correctly.